vignettes/download_jira_data.Rmd

```diff
@@ -49,7 +48,7 @@ save_path_issue_tracker_issue_comments <- conf[["issue_tracker"]][["jira"]][["is
 
 In the project configuration file, we define the domain which JIRA is hosted. The example configuration file uses the domain for Apache Software Foundation. We can use this information to queue the domain to identify all project keys available in its JIRA:
 
-```{r eval = FALSE}
+```{r}
```

Comment:
You should use git diff before sending the commit so you can spot these. Note you are modifying the original Notebook on parts that should not be modified. This is likely from when I showed how the Notebook works.  Regardless, there should be no reason for the Notebook to be added to your commit, since you are not changing that, right?